activerecord/test/cases/attribute_methods_test.rb

```diff
@@ -458,6 +474,53 @@ def topic.title() "b" end
     end
   end
 
+  test "user defined date attribute predicate" do
```

Comment:
I think we should have changed `user defined` -> `user-defined` in test cases as well if that matters.

Reply:
I've already addressed in 260a3c65122a726eea20828d4909e15502751c39, but either is fine to me since we already have some `user defined` in the repo.